charts/sentry-apps/values.yaml

```diff
@@ -0,0 +1,64 @@
+---
+spec:
+  metadata:
+    # spec.metadata.namespace -- namespace for the ArgoCD application
+    namespace: argocd
+  destination:
+    # spec.destination.server -- Kubernetes cluster for Sentry
+    server: "https://kubernetes.default.svc"
+    # spec.destination.namespace -- namespace for Sentry
+    namespace: sentry
+  source:
+    # spec.source.repoURL -- Chart museum to get Sentry
+    repoURL: "https://sentry-kubernetes.github.io/charts"
+    # spec.source.targetRevision -- revision of the chart to use for Sentry
+    targetRevision: 4.7.0
+
+user:
+  # user.create -- if true, creates the user defined by email and password.
+  create: true
+  # user.email -- Admin user email
+  email: admin@sentry.local
+  # user.password -- Admin user password
+  password: admin
+
+ingress:
+  # ingress.hostname -- Set this to enable the ingress configuration.
+  hostname: ""
```

Comment:
Maybe the documentation is not clear enough?

Reply:
I'll keep using `enabled`

Reply:
Yeah, enabled has to be enough since we only want to enforce it when it is enabled.

Regardings the docs, do we want to add a more complete example with stuff like an acme annotation?

The example could live in the ci/ subdir or in an examples/ subdir of it's own. It could even be a comment in the helm-docs that just tells people where to look in the upstream sentry chart.

Reply:
> do we want to add a more complete example with stuff like an acme annotation?

My point of view is that we should start using this first.